src/engraving/libmscore/measure.cpp

```diff
@@ -4083,8 +4083,7 @@ static bool hasAccidental(Segment* s)
 
 Fraction Measure::minSysTicks()
 {
-    Fraction minTicks = Fraction(1, 8);
-    //System* sys = system();
+    Fraction minTicks = Fraction(10, 1);
```

Comment:
That's pretty larg for a minimum, 10 whole notes?

Reply:
That is just the initialization of the variable. It needs to start a random high value in order for the function to correcly return the shortest note. The 1/8 was a mistake.

Reply:
Might be worth a comment in the code

Reply:
And the largest possible fraction is a longa `fraction(4, 1) or `DurationType::maxtype.ticks()` (?), I believe, seems a good starting point?

Reply:
Hmm, that could be quatdouple dotted, so even longer, makes it 39/4 is my math isn't failing me?

Reply:
Surprisingly close to your 10/1 ;-)

Reply:
Actually my math was failing me (as usual :-( ): it is 29/4.

Reply:
That starting point has *no effect whatsoever*. It just needs to be larger than any note you may encounter. It could just as well be 100 whole notes.

Reply:
True, still worth a comment to explain that IMHO

Reply:
the same here:
- make a named constexpr variable, make sure that you put a comment explaining these numbers